packages/mattermost-redux/src/client/client4.ts

```diff
@@ -870,7 +870,7 @@ export default class Client4 {
         );
     };
 
-    getProfilePictureUrl = (userId: string, lastPictureUpdate: number) => {
+    getProfilePictureUrl = (userId: string, lastPictureUpdate?: number) => {
```

Comment:
Using `UserProfile` above will also let you revert this change

Reply:
Hi @hmhealey  I already fix it.